imagery/i.signatures/i.signatures.html

```diff
@@ -0,0 +1,43 @@
+<h2>DESCRIPTION</h2>
+
+<em>i.signatures</em> module allows to manage signature files generated
+by <a href="i.gensig.html">i.gensig</a>,
+<a href="i.gensigset.html">i.gensigset</a> and
+<a href="i.svm.train.html">i.svm.train</a>.
+The module can perform multiple actions per run. The order of execution
+is "copy", "remove", "rename". If no actions are specified, then the module
+lists (prints) all signature files of specified type.
+
+<h2>EXAMPLES</h2>
+
+Print names of all signature files:
+<div class="code"><pre>
+i.signatures
+</pre></div>
+
+Print only signature files of certain type as a JSON:
+<div class="code"><pre>
+i.signatures type=sigset format=json
+</pre></div>
+
+Delete signature file called "foo" of type "sig" (i.gensig / i.maxlik).
+<div class="code"><pre>
+i.signatures remove=foo type=sig
+</pre></div>
+
+Copy signature file "bar" from mapset "baz" to current mapset
+<div class="code"><pre>
+i.signatures copy=bar@baz,best_version type=sigset
```

Comment:
The copy=from,to really makes more sense for g.copy which copies many different things, but it is not necessarily the best example of an interface. In Bash it does not matter much. In Python, ends up as awkward:

```python
gs.run_command("i.signatures", copy=(old_name, new_name), type="sigset")  
```

As opposed to:

```python
gs.run_command("i.signatures", from=old_name, to=new_name, type="sigset")  
```

Maybe we can discuss in-person.

Reply:
As this tool is not expected to be used too often, we'll keep it as is as each of alternatives has its downsides.